EventFiltering/PWGHF/HFFilter.cxx

```diff
@@ -235,6 +244,48 @@ struct HfFilter { // Main struct for HF triggers
       LOG(fatal) << "Can't apply optimisation if ML is not applied.";
     }
   }
+
+  /// compute TPC postcalibrated nsigma based on calibration histograms from CCDB
+  /// \param objectMean calibration histograms of mean from CCDB
+  /// \param objectSigma calibration histograms of sigma from CCDB
+  /// \param track is the track
+  /// \param chargePar is the PID species
+  /// \return the corrected Nsigma value for the PID species
+  template <typename P, typename T>
+  double GetPostTPCCalib(const P& objectMean, const P& objectSigma, const T& track, const int chargePar)
+  {
+    TH3F* calibMean = objectMean;
+    TH3F* calibSigma = objectSigma;
```

Comment:
Why is this needed? Can you not use `objectMean` and `objectSigma` directly?

Reply:
> Why is this needed? Can you not use `objectMean` and `objectSigma` directly?

Yes, you are right! I know, I know, I hate myself. Thanks a lot!